libs/service-portal/documents/src/components/DocumentLine/DocumentLineV3.tsx

```diff
@@ -0,0 +1,383 @@
+import cn from 'classnames'
+import format from 'date-fns/format'
+import { FC, useEffect, useRef, useState } from 'react'
+
+import {
+  DocumentV2,
+  DocumentV2Action,
+  DocumentV2Content,
+} from '@island.is/api/schema'
+import { Box, Icon, LoadingDots, Text, toast } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { ConfirmationModal, m } from '@island.is/service-portal/core'
+import { dateFormat } from '@island.is/shared/constants'
+import {
+  matchPath,
+  useLocation,
+  useNavigate,
+  useParams,
+} from 'react-router-dom'
+import { useDocumentList } from '../../hooks/useDocumentListV3'
+import { useIsChildFocusedorHovered } from '../../hooks/useIsChildFocused'
+import { useMailAction } from '../../hooks/useMailActionV2'
+import { DocumentsPaths } from '../../lib/paths'
+import { useDocumentContext } from '../../screens/Overview/DocumentContext'
+import { useGetDocumentInboxLineV3LazyQuery } from '../../screens/Overview/Overview.generated'
+import { messages } from '../../utils/messages'
+import { FavAndStashV3 } from '../FavAndStash/FavAndStashV3'
+import UrgentTag from '../UrgentTag/UrgentTag'
+import AvatarImage from './AvatarImage'
+import * as styles from './DocumentLine.css'
+
+interface Props {
+  documentLine: DocumentV2
+  img?: string
+  setSelectLine?: (id: string) => void
+  active: boolean
+  asFrame?: boolean
+  includeTopBorder?: boolean
+  selected?: boolean
+  bookmarked?: boolean
+}
+
+export const DocumentLineV3: FC<Props> = ({
+  documentLine,
+  img,
+  setSelectLine,
+  active,
+  asFrame,
+  includeTopBorder,
+  bookmarked,
+  selected,
+}) => {
+  const [hasFocusOrHover, setHasFocusOrHover] = useState(false)
+  const [hasAvatarFocus, setHasAvatarFocus] = useState(false)
+  const [modalData, setModalData] = useState<{
+    title: string
+    text: string
+  } | null>(null)
+  const [isModalVisible, setModalVisible] = useState(false)
+
+  const { formatMessage, lang } = useLocale()
+  const navigate = useNavigate()
+  const location = useLocation()
+  const date = format(new Date(documentLine.publicationDate), dateFormat.is)
```

Comment:
Could this return "Invalid Date"?

Reply:
Dont think so, it has been like this for ages in the older version